custom-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -0,0 +1,325 @@
+                  "title": "URL SharePoint"
+                },
+                "teamsUrl": {
+                  "type": "string",
+                  "description": "Teams URL",
+                  "title": "URL for teams"
+                },
+                "alreadyExists": {
+                  "type": "boolean",
+                  "description": "alreadyExists",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "enum": [
+                    true,
+                    false
+                  ]
+                },
+                "language": {
+                  "type": "string",
+                  "description": "language",
+                  "title": "",
+                  "x-ms-visibility": "internal"
+                },
+                "createTeam": {
+                  "type": "boolean",
+                  "description": "Create a teams group page",
+                  "title": "Create a team",
+                  "enum": [
+                    true,
+                    false
+                  ]
+                },
+                "approvalFlows": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "",
+                    "x-ms-visibility": "internal"
+                  },
+                  "description": "approvalFlows"
+                }
+              }
+            }
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "summary": "Group approval",
+        "operationId": "GroupMgrGroupApproval",
+        "description": "Will send an approval back to GroupMgr for the created group.",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Approved": {
+                  "type": "boolean",
+                  "description": "Group approval state.",
+                  "title": "Approved",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "x-ms-visibility": "important"
+                }
+              },
+              "required": [
+                "Approved"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+	{
+	  "propertyName": "Website",
+	  "propertyValue": "https://groupmgr.com/"
+	},
+	{
+	  "propertyName": "Privacy policy",
+	  "propertyValue": "https://groupmgr.com/privacy"
+	},
+	{
+	  "propertyName": "Categories",
+	  "propertyValue": "IT Operations;Productivty"
+	}
+   ],
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API-sleutel": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "vra-api-key"
+    }
+  },
+  "security": [
+    {
+      "API-sleutel": []
```

Comment:
@Jwaegebaert Please remove non-English characters from all the artifacts of connector